src/scope.h

```diff
@@ -113,7 +113,9 @@ struct Scope
     int explicitProtection;     // set if in an explicit protection attribute
 
     StorageClass stc;           // storage class
-    char *depmsg;               // customized deprecation message
+
+    // customized deprecation message
+    DeprecatedDeclaration *depdecl;
```

Comment:
There's not yet proper style guide to be followed by us. So your argument is still a personal opinion.
I have a free not to follow your comment, and you have a free not to fix the bug.

(And there's a third way - you can open another PR with the style you favor.)